pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -441,6 +441,34 @@ def test_timedelta(self, freq):
         tm.assert_index_equal(result1, result4)
         tm.assert_index_equal(result2, result3)
 
+    def test_timedelta_nat_comparisons(self):
```

Comment:
i think we have these tests in test_missing

Reply:
I couldn't find anything similar, but there are a lot of `test_missing.py` files.

Reply:
This would go in tests.scalar.timedeltas.test_arithmetic

Reply:
I've moved the test to `tests.scalar.timedeltas.test_arithmetic`, but there weren't any tests like it there so I added a new class for this test category.  If you want it to be ran outside of a class just let me know.